### ISSUE TITLE
Improve accessibility of Transcriber Registration page

### ISSUE THREAD

**Issue by AlOneill** [FreeUKGen/FreeBMD2#374]
The Transcriber Registration page has a banner-box that needs improving for the sake of screen-reader users — it appears to have slipped through previous revisions.

The current markup should be replaced with the following (apologies for lack of indentation):

`<div class="island island--navy">`
`<h2 class="banner__title">Questions? Data?</h2>`
`<p>For more information before you register, or if you have some data to contribute, please <a href="/contacts/new">Contact Us</a>, selecting the &lsquo;Volunteer&rsquo; option in that form.</p>`
`</div>`
